oap-server/generated-analysis/src/main/resources/official_analysis.oal

```diff
@@ -79,4 +79,15 @@ database_access_p99 = from(DatabaseAccess.latency).p99(10);
 database_access_p95 = from(DatabaseAccess.latency).p95(10);
 database_access_p90 = from(DatabaseAccess.latency).p90(10);
 database_access_p75 = from(DatabaseAccess.latency).p75(10);
-database_access_p50 = from(DatabaseAccess.latency).p50(10);
\ No newline at end of file
+database_access_p50 = from(DatabaseAccess.latency).p50(10);
+
+// CLR instance metric
+instance_clr_cpu = from(ServiceInstanceCLRCPU.usePercent).doubleAvg();
+instance_clr_gen0_collect_count = from(ServiceInstanceCLRGC.gen0CollectCount).sum();
+instance_clr_gen1_collect_count = from(ServiceInstanceCLRGC.gen1CollectCount).sum();
+instance_clr_gen2_collect_count = from(ServiceInstanceCLRGC.gen2CollectCount).sum();
+instance_clr_heap_memory = from(ServiceInstanceCLRGC.heapMemory).longAvg();
+instance_clr_available_completion_port_threads = from(ServiceInstanceCLRThread.availableCompletionPortThreads).count();
```

Comment:
The count of thread number? I think should you want new indicator? `max` or somthing?

Reply:
Same concern about the following thread related metric.

Reply:
This is an indicator of the CLR thread pool. Used to observe available I/OCP threads and worker threads  https://docs.microsoft.com/zh-cn/dotnet/api/system.threading.threadpool.getavailablethreads?view=netstandard-2.0

Reply:
Yes, I know roughly what it is. I just say, the indicator is not right.